test/support/cluster/orchestrator.rb

```diff
@@ -36,6 +40,18 @@ def down
     reset_cluster(@clients)
   end
 
+  def fail_serving_master
+    master, slave = take_replication_pairs(@clients)
+    master.shutdown
+    max_attempts = 600
```

Comment:
`600` seem excessive, again regarding CI time.

Reply:
tried with 10 and 100, the tests get flakey as cluster sometimes takes longer to recover (depending on `cluster-node-timeout` etc). Changing it to `500`, it shouldn't be a big problem since we return as soon as the failover is achieved.